common4j/src/main/com/microsoft/identity/common/java/platform/AbstractDevicePopManager.java

```diff
@@ -986,6 +993,22 @@ private String mintSignedHttpRequestInternal(@Nullable final String httpMethod,
             exception = e;
             errCode = KEYSTORE_NOT_INITIALIZED;
         } catch (final JOSEException e) {
+            if ((e != null && isNegativeInternalError(e)) ||
```

Comment:
`e` should never be null here

```suggestion
            if ((isNegativeInternalError(e)) ||
```

Reply:
ack